src/components/Review.js

```diff
@@ -24,7 +26,39 @@ function Reviews() {
     });
   }, []);
 
-  if (isLoading) {return (<p>Loading...</p>)}
+  const upvote = () => {
+    if (vote === false) {
+      voteReviewFunc(review_Id, 1)
```

Comment:
there's a delay when clicking the button, because you're making your patch before the optimistic rendering; suggest making the local change first, then doing the patch ... see ncnotes for example of this in action